packages/wallet/backend/src/rafiki/rafiki-client.ts

```diff
@@ -212,6 +212,11 @@ export class RafikiClient implements IRafikiClient {
       if (response.withdrawEventLiquidity?.message === 'Transfer exists') {
         return true
       }
+
+      if (response.withdrawEventLiquidity?.message === 'Invalid id') {
+        this.deps.logger.debug(`Nothing to withdraw for event ${eventId}`)
+        return true
+      }
```

Comment:
Instead of checking for the error message, we need to check the `balance` property that we receive on the webhook. If `balance` is 0, we do not need to call the `withdrawEventLiquidity` mutation.